src/Service/PaymentMethodService.php

```diff
@@ -141,6 +147,8 @@ public function getMethodsForCheckout()
         }
         $countryCode = Tools::strtolower($context->country->iso_code);
         $unavailableMethods = [];
+        $lang = $this->languageService->getLang();
```

Comment:
can we have getLang which accepts optional id parameter ? Inside getLang you could take translation by that id